coffee_lib/src/plugin.rs

```diff
@@ -149,7 +149,10 @@ impl Plugin {
     }
 
     /// upgrade the plugin to a new version.
-    pub async fn upgrade(&mut self) -> Result<(), CoffeeError> {
+    pub async fn upgrade(
+        &mut self, 
+        verbose: bool
```

Comment:
if the verbose it is not used, write it in this way `_: bool`

Reply:
https://github.com/coffee-tools/coffee/pull/217/commits/3291a453d3d1c62684fe3fb41925ca188408339c